sonic_data_client/mixed_db_client.go

```diff
@@ -75,6 +75,69 @@ type MixedDbClient struct {
 
 var mixedDbClientMap = map[string]MixedDbClient{}
 
+func getDpuAddress(dpuId string) (string, error) {
+	// Find DPU address by DPU ID from CONFIG_DB
+	// Design doc: https://github.com/sonic-net/SONiC/blob/master/doc/smart-switch/ip-address-assigment/smart-switch-ip-address-assignment.md?plain=1
+
+	var configDbConnector = swsscommon.NewConfigDBConnector()
+	configDbConnector.Connect(false)
+
+	// get bridge plane
+	bridgePlane, err := configDbConnector.Hget("MID_PLANE_BRIDGE", "GLOBAL", "bridge");
```

Comment:
>Hget [](http://example.com/codeflow?start=39&length=4)

Pending on another PR: [https://github.com/sonic-net/sonic-swss-common/pull/862](https://github.com/sonic-net/sonic-swss-common/pull/862)

Reply:
Fixed, Hget removed, this PR need another PR merge first.